src/Monitor/Monitor/help/Get-AzScheduledQueryRule.md

```diff
@@ -0,0 +1,136 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll-Help.xml
+Module Name: Az.Monitor
+online version: https://docs.microsoft.com/en-us/powershell/module/az.monitor/get-azscheduledqueryrule
+schema: 2.0.0
+---
+
+# Get-AzScheduledQueryRule
+
+## SYNOPSIS
+Gets Scheduled Query Resources
+
+## SYNTAX
+
+### BySubscriptionOrResourceGroup (Default)
+```
+Get-AzScheduledQueryRule [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ByRuleName
+```
+Get-AzScheduledQueryRule -ResourceGroupName <String> -RuleName <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Get-AzScheduledQueryRule -ResourceId <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Gets Scheduled Query Resources
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Parameter Set: BySubscriptionOrResourceGroup
```

Comment:
@shpimpal you can remove these `Parameter Set` comments made in this example and have multiple examples that display each of the different parameter sets

Reply:
Sorry, I didn't understand. There are examples for each parameter set.

Reply:
@shpimpal rather than putting an example for each parameter set all into one example, you'll create separate examples in separate headers for each parameter set. It would look like the following:

```
## EXAMPLES

### Example 1 - List by resource group

```powershell
PS C:\> Get-AzScheduledQueryRule -ResourceGroupName "Rac46PostSwapRG"
    ```

### Example 2 - Get by name

```powershell
PS C:\> Get-AzScheduledQueryRule -ResourceGroupName "Rac46PostSwapRG" -Name "logalertfoo"
    ```

### Example 3 - Get by resource id

```powershell
PS C:\> Get-AzScheduledQueryRule -ResourceId "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/Rac46PostSwapRG/providers/microsoft.insights/scheduledQueryRules/logalertfoo"
    ```
```

_Note_: the set of three backticks (` ``` `) at the end of every code block have an indentation to them for this comment -- there should be no indentation on the line with those backticks.

Reply:
Done.